pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
Well I guess it depends on the user. But yeah, potentially we can keep only adaptive. I need to add code to check the proper driver support.

> Yes, but what about triple buffering?
3B is another story. Instead to block the GPU when you miss the display interrupt. You can start the rendering into another 3rd buffer. Then you have 2 possibilities if you're still suck. Either you drop a frame (GL) or your stall the GPU (DX).

It is more complicated to implement and it still have various limitation. Potentially it could work well enough for us when game uses interlacing because we have a light frame after the heavy frame. So we have 2 vsync to render a frame. Adaptive is much easier to implement, just need to call SwapInterval with -1.

> `window!=monitor`
You mean size of the window different of size of monitor. I don't know how to query the size of the monitor. Besides what will happen in the next driver upgrade. What must happen for a 4:3 surface on a 16:9 monitor.